src/main/java/TextView.java

```diff
@@ -12,6 +12,8 @@
 public class TextView implements Observer
 {
 	final Logger logger = LoggerFactory.getLogger(TextView.class);
+	final static String NEWLINE = "\n";
+	final static String SPACE = "   ";
```

Comment:
- [ ] Add `private` keyword to the two Strings declaration
- [ ] Add `private` and `static` keywords to the logger declaration